src/pages/Validators/DelegatableValidatorsTable.tsx

```diff
@@ -0,0 +1,3 @@
+export default function DelegatableValidatorsTable() {
```

Comment:
Is this used anywhere? Can we remove it?

Reply:
interesting, it was deleted before, but sometimes when i perform git commands, some of the files will come back hahaha. i guess git learning is WIP!

Reply:
haha yeah I'm also learning